src/OpenLoco/Audio/VehicleChannel.h

```diff
@@ -4,16 +4,15 @@
 
 namespace OpenLoco::Audio
 {
-    class VehicleChannel
+    class VehicleChannel : public Channel
```

Comment:
In my opinion its best to move out vehicle specific code out entirely, the audio code should be unaware of such things, the vehicle only updates the channel parameters so it's probably best to have that with the vehicle code and not here.

Reply:
agreed, I'll look into it